proto/p4/config/v1/p4info.proto

```diff
@@ -46,6 +46,16 @@ message Documentation {
   string description = 2;
 }
 
+// A parsed representation of the annotation body.
+// Ex. empty annotation:           @my_anno1()
+// Ex. expression list annotation: @my_anno2(1+1, "Hello", 1 << 3)
+// Ex. KV-pair annotation:         @my_anno3(a=b,c=d)
+// Ex. Cumulative KV-pair          @my_anno3(e=f)  // adds to my_anno3's properties
+message AnnotationBody {
+  repeated string expressions = 1;
+  map <string,string> properties = 2;
```

Comment:
could we leave a space after the comma for all the maps (just like we did for the `P4TypeInfo` message)?

Reply:
Done.